tests/glib_api_test.cpp

```diff
@@ -0,0 +1,375 @@
+/*
+ * tests/glib_api_test.cpp
+ *
+ * Tests for the "wobbly" spring model, GObject API.
+ *
+ * See LICENCE.md for Copyright information.
```

Comment:
This file does not exist. `LICENSE` does on the other hand.

Reply:
To be honest, I should just replace all these with actual copyright headers, since this is now an @endlessm project